Sources/Core/Microsoft.StreamProcessing/StringHandling/MultiString.cs

```diff
@@ -708,9 +708,15 @@ public unsafe ColumnBatch<long> Contains2(string str, ColumnBatch<long> inBV, bo
             var largestr = this.col.charArray.contentString;
 
             for (int i = 0; i < this.Count; i++)
+            {
                 if ((bv[i >> 6] & (1L << (i & 0x3f))) == 0)
+                {
                     if (!(largestr.IndexOf(str, startscol[i] + 2, lengthscol[i], StringComparison.Ordinal) >= 0))
+                    {
                         rbv[i >> 6] |= (1L << (i & 0x3f));
+                    }
+                }
+            }
```

Comment:
This change hurts my eyes, my heart, and for some reason my spleen. I get it for consistency with the rule, but it really looked good before. :-(